### ISSUE TITLE
Python requirements aren't updated in PyPi package

### ISSUE THREAD

**Issue by jpweytjens** [deanmalmgren/textract#262]
#### Description
The git repository has the following python requirements in `requirements/python`

```
argcomplete==1.8.2
chardet==3.0.4
python-pptx==0.6.6
#pdfminer.six <-- go back to this after the shebang fix is released (see https://github.com/goulu/pdfminer/issues/27)
https://github.com/goulu/pdfminer/zipball/e6ad15af79a26c31f4e384d8427b375c93b03533#egg=pdfminer.six
docx2txt==0.6
beautifulsoup4==4.6.0
xlrd==1.0.0
EbookLib==0.16
SpeechRecognition==3.7.1
https://github.com/mattgwwalker/msg-extractor/zipball/master
six==1.10.0
pocketsphinx==0.1.3
```

yet the tar.gz file hosted at [PyPi](https://pypi.org/project/textract/) still has the following requirements

```
argcomplete==1.8.2
chardet==2.3.0
python-pptx==0.6.5
#pdfminer.six <-- go back to this after the shebang fix is released (see https://github.com/goulu/pdfminer/issues/27)
https://github.com/goulu/pdfminer/zipball/e6ad15af79a26c31f4e384d8427b375c93b03533#egg=pdfminer.six
docx2txt==0.6
beautifulsoup4==4.5.3
xlrd==1.0.0
EbookLib==0.15
SpeechRecognition==3.6.3
https://github.com/mattgwwalker/msg-extractor/zipball/master
six==1.10.0
pocketsphinx==0.1.3
```

#### To reproduce
Run `pip install textract` or `poetry add textract`

#### Tasks
- [ ] Update requirements for textract 1.6.1

**Comment by PifPof73** [comment]
You can try
`pip install --upgrade --nodeps textract`
which will ignore the dependicies and just try to install the package.

If one requirement is missing you should get an error message and than install this requirement first.

**Comment by JoelMon** [comment]
The problem I seem to be having while using `pipenv install textract` with Python 3.7 is that textract wants pocketsphinx==0.1.3 but pip wants to install pocketsphinx==0.1.15.

```

An error occurred while installing pocketsphinx==0.1.3 --hash=sha256:2b06fea365b55a40a0e51510b1b62a1575f264a38516ad20b940c133d569f6ef --hash=sha256:42c82f117009a7355f281932a5fe6bee1f68a432631712241c60d207ef267931 --hash=sha256:59f6de8ec032dccf059e0c5c9f1a21df553b9cfb6f4d5ebcfe580634c849f376 --hash=sha256:919503baba39604d6f1b8017a8fc239c41f77c7646748a0cc077001579a0bade --hash=sha256:9f932d7a6e2ec04ea8f0b019537fa0c92e702a226a02007223c55a46606c9568 --hash=sha256:ac125aadefc85ab78706a4d18ae66c3dcf89eb9e2a0cbd88c7a6086df420346d --hash=sha256:ccdc5fb5591649cd0edcdde541bc813b5f74bf5dd04872b937e551732eaa6847 --hash=sha256:f6bfa8e9a08026c46061364f59e9af8b0f4049142d4f155d9f07229d9b2d8281 --hash=sha256:fdbf33d926aece1c56fc7772fe7763f371eb8b0029ed85b498558aba15a8eb21! Will try again.
```

**Comment by DanAtShenTech** [comment]
@JoelMon you could do `pipenv graph` to see if anything else is calling for a higher version of pocketsphinx.

**Comment by JoelMon** [comment]
@DanAtShenTech you are right. Looks like pocketsphinx isn't installing. 

When I remove the enviroment, install a new one, and try to `pipenv install pocketsphinx` I get the following error:

```
Installing dependencies from Pipfile.lock (dc08dd)…
An error occurred while installing pocketsphinx==0.1.15 --hash=sha256:1c2a8cc7f2032f9e8214f5e4e864511fe37bfd27f2045fbf8aed9c54569dc881 --hash=sha256:1f3eaca1b49d579e89e909bb101ad033493a1c9dddd8c72afd6b755c70365d56 --hash=sha256:34d290745c7dbe6fa2cac9815b5c19d10f393e528ecd70e779c21ebc448f9b63 --hash=sha256:3f438f9f5729a7629c122784697884cff5f2a2e022aef270a4081985d15eae88 --hash=sha256:41d2107b74597cf8444f9e36b91363d60097b4fa88a9c750760a85209deef439 --hash=sha256:667152b4139ba46de172fa627404fe284b2ecd5d6bfcfe905b068801324ba7fb --hash=sha256:71e7cf04433615e21ad3c28a2cc2daa43e96e570f2611f142eb3468f4f7789a6 --hash=sha256:72286c3ecaa6641d426ca80bd4d1b3241c26b1703af419429ef779719e007443 --hash=sha256:855c6761d008cdb4fc2d9aded5c1a0f163ce901f8b64075d36a88ae7814af755 --hash=sha256:8ca14694d7cc111796e63c13c61420553a43a29c63c386c975bfc213d8da3e55 --hash=sha256:9b477fcf8829f53ab470fdcf3fa77db00d71eee321353d10b306f0e5b95e8591 --hash=sha256:bfbbeddb20196abbb04b8f7b5dc03628c7df0dc87e760326d76d37b27fb9d6f9 --hash=sha256:d20b497369bd108dd135b3d4eed1d3edd2c8b51b1b696e67600adbfff2fe3316 --hash=sha256:e2a4f22cddec68e75e796068ae82095dd8dba1187aa3a8abe305bdc6c565e8b4 --hash=sha256:f0fdfe162bc99590e8666251a55d6311ac4cb137482a871f93d38b1834251253! Will try again.
  🐍   ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 1/1 — 00:00:02
Installing initially failed dependencies…
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/dist-packages/pipenv/core.py", line 1992, in do_install
[pipenv.exceptions.InstallError]:       skip_lock=skip_lock,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/dist-packages/pipenv/core.py", line 1253, in do_init
[pipenv.exceptions.InstallError]:       pypi_mirror=pypi_mirror,
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/dist-packages/pipenv/core.py", line 859, in do_install_dependencies
[pipenv.exceptions.InstallError]:       retry_list, procs, failed_deps_queue, requirements_dir, **install_kwargs
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/dist-packages/pipenv/core.py", line 763, in batch_install
[pipenv.exceptions.InstallError]:       _cleanup_procs(procs, not blocking, failed_deps_queue, retry=retry)
[pipenv.exceptions.InstallError]:   File "/usr/local/lib/python3.6/dist-packages/pipenv/core.py", line 681, in _cleanup_procs
[pipenv.exceptions.InstallError]:       raise exceptions.InstallError(c.dep.name, extra=err_lines)
[pipenv.exceptions.InstallError]: ['Collecting pocketsphinx==0.1.15 (from -r /tmp/pipenv-oxj615vw-requirements/pipenv-zl0zt732-requirement.txt (line 1))', '  Using cached https://files.pythonhosted.org/packages/cd/4a/adea55f189a81aed88efa0b0e1d25628e5ed22622ab9174bf696dd4f9474/pocketsphinx-0.1.15.tar.gz', 'Building wheels for collected packages: pocketsphinx', '  Building wheel for pocketsphinx (setup.py): started', "  Building wheel for pocketsphinx (setup.py): finished with status 'error'", '  Running setup.py clean for pocketsphinx', 'Failed to build pocketsphinx', 'Installing collected packages: pocketsphinx', '  Running setup.py install for pocketsphinx: started', "    Running setup.py install for pocketsphinx: finished with status 'error'"]
[pipenv.exceptions.InstallError]: ['ERROR: Complete output from command /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3.7m -u -c \'import setuptools, tokenize;__file__=\'"\'"\'/tmp/pip-install-2k9k10aq/pocketsphinx/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' bdist_wheel -d /tmp/pip-wheel-o9xnwqfm --python-tag cp37:', '  ERROR: running bdist_wheel', '  running build_ext', "  building 'sphinxbase._sphinxbase' extension", '  swigging deps/sphinxbase/swig/sphinxbase.i to deps/sphinxbase/swig/sphinxbase_wrap.c', '  swig -python -modern -threads -Ideps/sphinxbase/include -Ideps/sphinxbase/include/sphinxbase -Ideps/sphinxbase/include/android -Ideps/sphinxbase/swig -outdir sphinxbase -o deps/sphinxbase/swig/sphinxbase_wrap.c deps/sphinxbase/swig/sphinxbase.i', "  unable to execute 'swig': No such file or directory", "  error: command 'swig' failed with exit status 1", '  ----------------------------------------', '  ERROR: Failed building wheel for pocketsphinx', '    ERROR: Complete output from command /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3.7m -u -c \'import setuptools, tokenize;__file__=\'"\'"\'/tmp/pip-install-2k9k10aq/pocketsphinx/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' install --record /tmp/pip-record-3nffc1dl/install-record.txt --single-version-externally-managed --compile --install-headers /home/user/.local/share/virtualenvs/Outline-H7ySTsce/include/site/python3.7/pocketsphinx:', '    ERROR: running install', '    running build_ext', "    building 'sphinxbase._sphinxbase' extension", '    swigging deps/sphinxbase/swig/sphinxbase.i to deps/sphinxbase/swig/sphinxbase_wrap.c', '    swig -python -modern -threads -Ideps/sphinxbase/include -Ideps/sphinxbase/include/sphinxbase -Ideps/sphinxbase/include/android -Ideps/sphinxbase/swig -outdir sphinxbase -o deps/sphinxbase/swig/sphinxbase_wrap.c deps/sphinxbase/swig/sphinxbase.i', "    unable to execute 'swig': No such file or directory", "    error: command 'swig' failed with exit status 1", '    ----------------------------------------', 'ERROR: Command "/home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3.7m -u -c \'import setuptools, tokenize;__file__=\'"\'"\'/tmp/pip-install-2k9k10aq/pocketsphinx/setup.py\'"\'"\';f=getattr(tokenize, \'"\'"\'open\'"\'"\', open)(__file__);code=f.read().replace(\'"\'"\'\\r\\n\'"\'"\', \'"\'"\'\\n\'"\'"\');f.close();exec(compile(code, __file__, \'"\'"\'exec\'"\'"\'))\' install --record /tmp/pip-record-3nffc1dl/install-record.txt --single-version-externally-managed --compile --install-headers /home/user/.local/share/virtualenvs/Outline-H7ySTsce/include/site/python3.7/pocketsphinx" failed with error code 1 in /tmp/pip-install-2k9k10aq/pocketsphinx/']
ERROR: ERROR: Package installation failed...
  ☤  ▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉▉ 0/1 — 00:00:02

```

**Comment by DanAtShenTech** [comment]
@JoelMon Not sure why pocketsphinx would not install for you.  Seems like if you can get that problem worked out, then you should be able to install textract.  Did you try exiting your `pipenv` environment and just doing a `pip install pocketsphinx`?  I run Ubuntu and had no problem with pocketsphinx being installed during installation of textract (but notice my [comment above](https://github.com/deanmalmgren/textract/issues/262#issuecomment-481252377) about how I ended up installing directly from the deanmalmgren/textract repo in order to remedy a chardet dependency problem).

**Comment by DanAtShenTech** [comment]
@JoelMon:  Incidentally, @deanmalmgren has not approved any pull requests to the textract repo for nearly two years.  There are [pull requests](https://github.com/deanmalmgren/textract/pulls) to update the pocketsphinx version dependency (as well as other library version dependencies), but none are being approved.  You might also see my comment about this matter [here](https://github.com/fourdigits/wagtail_textract/issues/22#issuecomment-483889981).

**Comment by JoelMon** [comment]
@DanAtShenTech 


So I went over to pocketsphinx and opened an issue cmusphinx/pocketsphinx#171. It was closed with the message:

> You need to install swig

I'm going to look for another package that does something similar to textract. I need to only extract the text of a PDF and parse it. Textract seems like it does a whole lot more than what I need. 


I tried your suggestions and I've posted the results bellow in case someone out there is searching for the error. Thank you for your help @DanAtShenTech, I appricate it.

> Did you try exiting your pipenv environment and just doing a pip install pocketsphinx?

I get the following error:

```
Collecting pocketsphinx
  Using cached https://files.pythonhosted.org/packages/cd/4a/adea55f189a81aed88efa0b0e1d25628e5ed22622ab9174bf696dd4f9474/pocketsphinx-0.1.15.tar.gz
Installing collected packages: pocketsphinx
  Running setup.py install for pocketsphinx ... error
    ERROR: Complete output from command /usr/bin/python3.6 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-04ssvg1b/pocketsphinx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-xpvi_lie/install-record.txt --single-version-externally-managed --compile:
    ERROR: /usr/lib/python3.6/distutils/dist.py:261: UserWarning: Unknown distribution option: 'long_description_content_type'
      warnings.warn(msg)
    running install
    running build_ext
    building 'sphinxbase._sphinxbase' extension
    swigging deps/sphinxbase/swig/sphinxbase.i to deps/sphinxbase/swig/sphinxbase_wrap.c
    swig -python -modern -threads -Ideps/sphinxbase/include -Ideps/sphinxbase/include/sphinxbase -Ideps/sphinxbase/include/android -Ideps/sphinxbase/swig -outdir sphinxbase -o deps/sphinxbase/swig/sphinxbase_wrap.c deps/sphinxbase/swig/sphinxbase.i
    unable to execute 'swig': No such file or directory
    error: command 'swig' failed with exit status 1
    ----------------------------------------
ERROR: Command "/usr/bin/python3.6 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-04ssvg1b/pocketsphinx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-xpvi_lie/install-record.txt --single-version-externally-managed --compile" failed with error code 1 in /tmp/pip-install-04ssvg1b/pocketsphinx/
```



> from the repo (not pipenv install...). This still installs into the environment (and not system-wide) since it is activated.

I get the following error:

```

user@user-Oryx-Pro ~/Documents/Development/Outline $ pipenv --python 3.6
Creating a virtualenv for this project…
Pipfile: /home/user/Documents/Development/Outline/Pipfile
Using /usr/bin/python3 (3.6.7) to create virtualenv…
⠴ Creating virtual environment...Using base prefix '/usr'
New python executable in /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3
Also creating executable in /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python
Installing setuptools, pip, wheel...
done.
Running virtualenv with interpreter /usr/bin/python3

✔ Successfully created virtual environment! 
Virtualenv location: /home/user/.local/share/virtualenvs/Outline-H7ySTsce
Creating a Pipfile for this project…
user@user-Oryx-Pro ~/Documents/Development/Outline $ pipenv shell
Launching subshell in virtual environment…
user@user-Oryx-Pro ~/Documents/Development/Outline $  . /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/activate
(Outline) user@user-Oryx-Pro ~/Documents/Development/Outline $ pip install git+https://github.com/deanmalmgren/textract.git –upgrade
ERROR: Invalid requirement: '–upgrade'

(Outline) user@user-Oryx-Pro ~/Documents/Development/Outline $ pip install git+https://github.com/deanmalmgren/textract.git
Collecting git+https://github.com/deanmalmgren/textract.git
  Cloning https://github.com/deanmalmgren/textract.git to /tmp/pip-req-build-vhz544qs
  Running command git clone -q https://github.com/deanmalmgren/textract.git /tmp/pip-req-build-vhz544qs
Collecting argcomplete==1.8.2 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/f0/0f/f965f1520e6ba24b63320919eecfbe3d03debd32402e0c61a08e8fa02d17/argcomplete-1.8.2-py2.py3-none-any.whl
Collecting chardet==3.0.4 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/bc/a9/01ffebfb562e4274b6487b4bb1ddec7ca55ec7510b22e4c51f14098443b8/chardet-3.0.4-py2.py3-none-any.whl (133kB)
     |████████████████████████████████| 143kB 2.8MB/s 
Collecting python-pptx==0.6.6 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/eb/8f/24cf444517959ebd46bda3eea176c5832f9dacec1556e7b5d8d61ee1a8a8/python-pptx-0.6.6.tar.gz (8.4MB)
     |████████████████████████████████| 8.4MB 9.0MB/s 
Collecting docx2txt==0.6 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/aa/72/f02730ec3b0219d8f783a255416339b02ff8b6a300c817abf0505833212a/docx2txt-0.6.tar.gz
Collecting beautifulsoup4==4.6.0 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/9e/d4/10f46e5cfac773e22707237bfcd51bbffeaf0a576b0a847ec7ab15bd7ace/beautifulsoup4-4.6.0-py3-none-any.whl (86kB)
     |████████████████████████████████| 92kB 3.9MB/s 
Collecting xlrd==1.0.0 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/0c/b0/8946fe3f9c2690c164aaa88dfd43b56347d3cdeac34124b988acd1aaa151/xlrd-1.0.0-py3-none-any.whl (143kB)
     |████████████████████████████████| 153kB 4.6MB/s 
Collecting EbookLib==0.16 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/6d/4b/d432e26fd14cc8ad1bf86453261115127931312c7fda3eb0911efe67bed4/EbookLib-0.16.tar.gz
Collecting SpeechRecognition==3.7.1 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/aa/99/18926cba2e18453756a90332d871541a26f8f9e81dbfc4bed0e7d3ffbf04/SpeechRecognition-3.7.1-py2.py3-none-any.whl (32.8MB)
     |████████████████████████████████| 32.8MB 1.7MB/s 
Collecting six==1.10.0 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/c8/0a/b6723e1bc4c516cb687841499455a8505b44607ab535be01091c0f24f079/six-1.10.0-py2.py3-none-any.whl
Collecting pocketsphinx==0.1.3 (from textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/93/5f/a968e5d53d25e32deb78c3e169fd8612ecf53cc76e32cb40e19be35696af/pocketsphinx-0.1.3.tar.bz2 (29.4MB)
     |████████████████████████████████| 29.4MB 7.3MB/s 
Collecting lxml>=3.1.0 (from python-pptx==0.6.6->textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/35/8a/5e066949f2b40caac32c7b2a77da63ad304b5fbe869036cc3fe4a198f724/lxml-4.3.3-cp36-cp36m-manylinux1_x86_64.whl (5.7MB)
     |████████████████████████████████| 5.7MB 6.3MB/s 
Collecting Pillow>=2.6.1 (from python-pptx==0.6.6->textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/d2/c2/f84b1e57416755e967236468dcfb0fad7fd911f707185efc4ba8834a1a94/Pillow-6.0.0-cp36-cp36m-manylinux1_x86_64.whl (2.0MB)
     |████████████████████████████████| 2.0MB 3.0MB/s 
Collecting XlsxWriter>=0.5.7 (from python-pptx==0.6.6->textract==1.6.1)
  Downloading https://files.pythonhosted.org/packages/35/7d/d432d6718452432b66e3ec29efafad2deea1bc9e5232951413523aeef8df/XlsxWriter-1.1.7-py2.py3-none-any.whl (139kB)
     |████████████████████████████████| 143kB 8.6MB/s 
Building wheels for collected packages: textract, python-pptx, docx2txt, EbookLib, pocketsphinx
  Building wheel for textract (setup.py) ... done
  Stored in directory: /tmp/pip-ephem-wheel-cache-re_s8v3c/wheels/9e/6d/a2/ec00a5d61838bec8e9e1ffdd53e14d5e833840d7fe7e9a413c
  Building wheel for python-pptx (setup.py) ... done
  Stored in directory: /home/user/.cache/pip/wheels/73/18/60/8f627102e871bfbe14f782827e9c1d65a7ef6530ff3722e2c6
  Building wheel for docx2txt (setup.py) ... done
  Stored in directory: /home/user/.cache/pip/wheels/a8/a1/2e/61cea06f166c6f6ac4df4d75a409328406bd0d2f8e007592f9
  Building wheel for EbookLib (setup.py) ... done
  Stored in directory: /home/user/.cache/pip/wheels/39/d0/4b/41174a759a76b84ff431345680180c6a283d85132ccd456525
  Building wheel for pocketsphinx (setup.py) ... error
  ERROR: Complete output from command /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-j11l2nzz/pocketsphinx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' bdist_wheel -d /tmp/pip-wheel-qiyr7t46 --python-tag cp36:
  ERROR: running bdist_wheel
  running build_ext
  building 'sphinxbase._ad' extension
  swigging swig/sphinxbase/ad.i to swig/sphinxbase/ad_wrap.c
  swig -python -modern -Ideps/sphinxbase/include -Ideps/sphinxbase/include/sphinxbase -Ideps/sphinxbase/include/android -Ideps/sphinxbase/swig -outdir sphinxbase -o swig/sphinxbase/ad_wrap.c swig/sphinxbase/ad.i
  unable to execute 'swig': No such file or directory
  error: command 'swig' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pocketsphinx
  Running setup.py clean for pocketsphinx
Successfully built textract python-pptx docx2txt EbookLib
Failed to build pocketsphinx
Installing collected packages: argcomplete, chardet, lxml, Pillow, XlsxWriter, python-pptx, docx2txt, beautifulsoup4, xlrd, six, EbookLib, SpeechRecognition, pocketsphinx, textract
  Running setup.py install for pocketsphinx ... error
    ERROR: Complete output from command /home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-j11l2nzz/pocketsphinx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-05s1uans/install-record.txt --single-version-externally-managed --compile --install-headers /home/user/.local/share/virtualenvs/Outline-H7ySTsce/include/site/python3.6/pocketsphinx:
    ERROR: running install
    running build_ext
    building 'sphinxbase._ad' extension
    swigging swig/sphinxbase/ad.i to swig/sphinxbase/ad_wrap.c
    swig -python -modern -Ideps/sphinxbase/include -Ideps/sphinxbase/include/sphinxbase -Ideps/sphinxbase/include/android -Ideps/sphinxbase/swig -outdir sphinxbase -o swig/sphinxbase/ad_wrap.c swig/sphinxbase/ad.i
    unable to execute 'swig': No such file or directory
    error: command 'swig' failed with exit status 1
    ----------------------------------------
ERROR: Command "/home/user/.local/share/virtualenvs/Outline-H7ySTsce/bin/python3 -u -c 'import setuptools, tokenize;__file__='"'"'/tmp/pip-install-j11l2nzz/pocketsphinx/setup.py'"'"';f=getattr(tokenize, '"'"'open'"'"', open)(__file__);code=f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' install --record /tmp/pip-record-05s1uans/install-record.txt --single-version-externally-managed --compile --install-headers /home/user/.local/share/virtualenvs/Outline-H7ySTsce/include/site/python3.6/pocketsphinx" failed with error code 1 in /tmp/pip-install-j11l2nzz/pocketsphinx/
```

**Comment by jpweytjens** [comment]
> 
> 
> @JoelMon: Incidentally, @deanmalmgren has not approved any pull requests to the textract repo for nearly two years. There are [pull requests](https://github.com/deanmalmgren/textract/pulls) to update the pocketsphinx version dependency (as well as other library version dependencies), but none are being approved. You might also see my comment about this matter [here](https://github.com/fourdigits/wagtail_textract/issues/22#issuecomment-483889981).

@DanAtShenTech, are you interested in maintaining textract with me as co-maintainer? I reached out to @deanmalmgren on Twitter as you saw. I'm interested to help out where I can, but I don't have the time or the experience to do it by myself. Glancing at the open issues, updating the dependencies and making textract install easily again seems like the main challenge. 

I've been able to get textract up and running with `poetry` as dependency manager and by installing `swig` as mentioned in this issue.

**Comment by jpweytjens** [comment]
Closed by #292
